txt2regex.sh

```diff
@@ -1206,7 +1206,11 @@ escCharList() {
     # shellcheck disable=SC1003
     if [ "$(getMeta "ax_${progs[$1]}" 6)" == '\' ]; then
         escape_metachar=$(getMeta "ax_${progs[$1]}" 4)
-        uin="${uin/\\/$escape_metachar$escape_metachar}"
+        if [[ ${BASH_VERSINFO[0]} -le 5 && ${BASH_VERSINFO[1]} -lt 2 ]]; then
```

Comment:
I'm ok with doing the version check in this case. But this logic has an issue, instead of only targeting versions <5.2, it excludes versions 3.2 and 4.2, for example, due to the second condition `${BASH_VERSINFO[1]} -lt 2`.

> You can see [in the CI run](https://github.com/aureliojargas/txt2regex/runs/8279046626?check_suite_focus=true), step "Run make test-bash", that versions 3.2 and 4.2 have failed because the quotes were added in the output. The CI being successful even with those failures is a bug that I've now fixed in 0cb32a7.

We **can** safely use the quotes since bash 4.3, and we **must** use them since bash 5.2. How about we simplify the logic to check only the major version, if it's older than 5?

```suggestion
        if [ ${BASH_VERSINFO[0]} -lt 5 ]; then
```

Reply:
Whoops, fixed now. Thanks.